website/docs/specification/DigitalProductPassport.md

```diff
@@ -307,194 +307,94 @@ Note - this sample describes the digital product passport payload only - ie the
-          }
+          "type": "MaterialProvenance",
+          "originCountry": "EU",
+          "materialType": "EPP",
+          "massFraction": 0.6,
+          "recycled": true,
+          "hazardous": false
         }
       ],
-      "name": "string"
-    },
-    "manufacturedAt": {
-      "identifiers": [
+      "conformityClaims": [
+        {
+          "type": "LinkRole",
+          "target": "https://supplier.example.com/material/reuse-certificate",
+          "linkRelationship": "untpConformity"
+        },
         {
-          "scheme": "http://example.com",
-          "identiferValue": "string",
-          "identifierURI": "http://example.com",
-          "verificationEvidence": {
-            "format": "w3c_vc",
-            "assuranceLevel": "Self",
-            "credentialReference": "http://example.com"
-          }
+          "type": "LinkRole",
+          "target": "https://supplier.example.com/manufacturing/carbon-emissions-certificate",
+          "linkRelationship": "untpConformity"
         }
       ],
-      "name": "string",
-      "location": "http://example.com",
-      "operatedBy": {
-        "identifiers": [
-          {
-            "scheme": "http://example.com",
-            "identiferValue": "string",
-            "identifierURI": "http://example.com",
-            "verificationEvidence": {
-              "format": "w3c_vc",
-              "assuranceLevel": "Self",
-              "credentialReference": "http://example.com"
-            }
-          }
-        ],
-        "name": "string"
-      }
-    },
-    "materialsProvenance": [
-      {
-        "originCountry": "string",
-        "materialType": {
-          "scheme": "http://example.com",
-          "classifierValue": "string",
-          "classifierName": "string",
-          "classifierURL": "http://example.com"
-        },
-        "massFraction": 0,
-        "recycled": true,
-        "hazardous": true
-      }
-    ],
-    "conformityClaims": [
-      {
-        "topic": "environment.energy",
-        "standardOrRegulation": "http://example.com",
-        "criteriaReference": "http://example.com",
-        "claimedValues": [
-          {
-            "name": "string",
-            "value": 0,
-            "accuracy": 0,
-            "unit": "string"
-          }
-        ],
-        "benchmarkValue": {
-          "name": "string",
-          "value": 0,
-          "accuracy": 0,
-          "unit": "string"
-        },
-        "benchmarkReference": "http://example.com",
-        "conformance": true,
-        "conformityEvidence": {
-          "format": "w3c_vc",
-          "assuranceLevel": "Self",
-          "credentialReference": "http://example.com"
+      "recyclingInstructions": "http://brand-owner.example.com/nordic-pioneer/recycling",
+      "traceabilityInformation": [
+        {
+          "type": "UNTPEvent",
+          "eventReference": "http://manufacturer.example.com/293847293847",
+          "eventType": "aggregation"
```

Comment:
Why eventType instead of just `type`?

Reply:
That's a great point, @Fak3! 
(Not sure I'll have time to update before Thursday - happy to accept a commit, tho)